lib/GameConstants.h

```diff
@@ -1296,14 +1297,17 @@ class Obstacle : public BaseForID<Obstacle, si32>
 	DLL_LINKAGE static Obstacle fromString(const std::string & identifier);
 };
 
-enum class ESpellSchool: ui8
+enum class ESpellSchool: int16_t

```

Comment:
Likely not a big deal, but was ui8 before. Might make sense to change to int8_t, not to int16_t

Reply:
Yes, it was ui8, but is int8_t enough to us?

Reply:
- if goal was to allow -1 as valid value then we should just switch to int8_t
- if goal is to ensure that any potential new elements will fit - then IMO all such identifiers should use int32_t, as majority of our existing identifiers do

That being said, highly doubt that we'll ever have 100+ schools

Reply:
done as int8_t